integration-tests/src/test/java/org/apache/druid/tests/query/ITSqlQueryTest.java

```diff
@@ -133,10 +143,10 @@ private void assertEquals(String expected, String actual)
     }
   }
 
-  private void assertEquals(int expected, int actual)
+  private void assertEquals(int expected, int actual, String message)
```

Comment:
Is this method really needed, can't we just use `Assert.assertEquals()`?

Reply:
It's needed for troubleshooting from the log to understand what's returned from server in case of test failure

Reply:
You're right, we can use testNG's Assert. 
Previously my IDE suggested me to use JUnit's Assert which is not part of the IT module, so I had to add this method